learn/work/calibration-splits/index.qmd

```diff
@@ -0,0 +1,478 @@
+---
+title: "Automatic calibration splits for resamples"
+categories:
+  - post-processing
+  - calibration
+  - rsample
+type: learn-subsection
+weight: 6
+description: | 
+  Learn how tidymodels generates automatic calibration sets for post-processing.
+toc: true
+toc-depth: 2
```

Comment:
```suggestion
toc-depth: 3
```

due to the length of the article i think upping the TOC depth is nice, otherwise we don't update the toc that fast